lib/vscode/product.json

```diff
@@ -20,9 +20,8 @@
 	"darwinBundleIdentifier": "com.visualstudio.code.oss",
 	"linuxIconName": "com.visualstudio.code.oss",
 	"licenseFileName": "LICENSE.txt",
-	"reportIssueUrl": "https://github.com/microsoft/vscode/issues/new",
+	"reportIssueUrl": "https://github.com/cdr/code-server/issues/new",
```

Comment:
This is probably a task for another time but I wonder if we should change these at build time instead of patching like we have been.